packages/firebase_in_app_messaging/firebase_in_app_messaging/android/src/main/java/io/flutter/plugins/firebase/inappmessaging/FirebaseInAppMessagingPlugin.java

```diff
@@ -30,26 +31,28 @@ public void onAttachedToEngine(FlutterPluginBinding binding) {
   }
 
   @Override
-  public void onDetachedFromEngine(FlutterPluginBinding binding) {
+  public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
     if (channel != null) {
       channel.setMethodCallHandler(null);
       channel = null;
     }
   }
 
   @Override
-  public void onMethodCall(MethodCall call, Result result) {
+  public void onMethodCall(MethodCall call, @NonNull Result result) {
     switch (call.method) {
       case "FirebaseInAppMessaging#triggerEvent":
         {
           String eventName = call.argument("eventName");
+          assert eventName != null;
```

Comment:
It feels weird to add this, but since we're the ones sending the argument I guess it's okay 🤔 
How is it displayed in the console when Flutter is running?

Reply:
It would be just for debug and never triggered as it is a required param on the universal instance API, I included to remove warning for unchecked cast. I've updated to what we use in other instances - `Objects.requireNonNull()`.